myuw/templates/handlebars/teaching/photo_list.html

```diff
@@ -2,103 +2,85 @@
-                        {{ /if }}
-                        <td>{{ email }}</td>
-                	</tr>
-                    {{ /each }}
-                    </tbody>
-                </table>
-
-
-                <!-- mobile view 
                 <div id="student_list">
-                    {{ #each sections.0.registrations }}
-                        <div id="student_{{ regid }}">
-                            <div style="float: left;">
-                                <img class="classlist-photo" src="/photo/{{ url_key }}">
-                            </div>
-                            <div style="float: left; padding-left: 10px; padding-bottom: 10px;">
-                                <strong>{{ surname }}, {{ name }}</strong><br />
-                                Student Number: {{ student_number }} <br />
-                                UW NetID: {{ netid }} <br />
-                                Credits: {{ credits }} <Br />
-                                Major: {{ #each majors }} {{ full_name }} {{ /each }}<br />
-                                Class: {{ class }} <br />
-                                Email: <a href="mailto:{{ email }}">{{ email }}</a> <br />
-                                {{ #if linked_sections }}
-                                {{ #each linked_sections }}
-                                    {{ ucfirst type }} Section: 
-                                    {{ #each sections }}
-                                        {{ this }}
-                                    {{ /each }}
-                                    <br />
-                                {{ /each }}
-                                {{ /if }}
-                            </div>
-                            <br style="clear: both;">
-                        </div>
-                    {{ /each }}
-                </div> -->
-        
+
+                    <ol id="student_sort">
+
+                        {{ #each sections.0.registrations }}
+                            <li id="student_{{ regid }}">
+
+                                <img class="classlist-photo myuw-photo-list-image" src="/photo/{{ url_key }}">
+
+                                <div class="myuw-photo-list-content">
+                                    <div>
+                                        <strong>{{ surname }}, {{ name }}</strong><br />
+                                        <span class="sr-only">Email:&nbsp;&nbsp;</span><a href="mailto:{{ email }}">{{ email }}</a>
+                                    </div>
+                                    <div class="myuw-photo-list-meta">
+                                        <div class="pull-left" style="width:50%; min-width: 272px;">
+                                        Student Number:&nbsp;&nbsp;{{ student_number }} <br />
+                                        UW NetID:&nbsp;&nbsp;{{ netid }} <br />
+                                        Major:&nbsp;&nbsp;{{ #each majors }} {{ full_name }} {{ /each }}<br />
```

Comment:
The space in here `{{ #each majors }} {{ full_name }} {{ /each }}` before full_name is causing 3 spaces after "UW NetID:"